wbooks/src/app/components/AppNavigator/index.tsx

```diff
@@ -18,6 +20,11 @@ function LibraryStackScreen() {
     <Stack.Navigator screenOptions={SCREEN_OPTS as object}>
       <Stack.Screen name={ROUTES.BookList} component={BookList} options={{ title: 'Listado de libros' }} />
       <Stack.Screen name={ROUTES.BookDetail} component={BookDetail} options={{ title: 'Detalle' }} />
+      <Stack.Screen
+        name={ROUTES.Search}
+        component={Search}
+        options={{ headerTitle: SearchBar, headerRight: undefined, headerLeft: () => null }}
```

Comment:
`options={{ headerTitle: () => <SearchBar /> }}`

Why `headerRight: undefined`?
I think `headerLeft: () => null` it's not correct

Reply:
![image](https://user-images.githubusercontent.com/45558924/106946062-61e8a280-6707-11eb-9585-43a52469972d.png)

Header Right is present because otherwise, the right magnifier keeps on showing up

Reply:
This is wrong, the search icon must be shown only on the `BookList` screen, not on all the screens of the stack